tests/app/main/test_services.py

```diff
@@ -2400,7 +2431,7 @@ def test_shows_confirmation_banner_and_button_when_copying_all(self, get_metadat
             "[normalize-space()='Are you sure you want to add all your cloud hosting services?']"
         )
         assert doc.xpath(
-            "//button[@class='button-save banner-action'][normalize-space()='Yes, add all services']"
+            "//button[@class='button-save banner-action'][normalize-space()='Yes, add']"
```

Comment:
`Yes, add` feels like weird content... Could it just be `Yes`?

Reply:
Going to talk to Jane about this as well as some other content stuff this afternoon.

Reply:
Cool. Drop me a line when that's all updated and I'll 👍